model/src/main/proto/profile.proto

```diff
@@ -73,6 +73,9 @@ message Profile {
 
   // Represents the number of times a user has successfully logged into their profile.
   int32 number_of_logins = 14;
+
+  // Determines if the continue button animation has been seen by the user or not.
```

Comment:
This needs a bit more context since it's at the profile level, not at the exploration level, contextually. Suggest:

```suggestion
  // Determines if the in-exploration continue button animation has been seen by the user or not.
```

Reply:
done